forms/uds/a1/form_a1_ivp_error_checks_p.csv

```diff
@@ -0,0 +1,52 @@
+error_code,error_no,error_type,form_name,packet,var_name,check_type,test_name,short_desc,full_desc,test_logic,comp_forms,comp_vars,do_in_redcap,in_prev_versions,questions
+a1-ivp-p-1001,1001,Alert,a1,I,RACEWHITE,Plausibility,"If one race is known, cannot check Unknown","If  RACEWHITE, RACEBLACK, RACEAIAN, RACENHPI, RACEASIAN, ETHISPANIC, or RACEOTH=1 then RACEUNKN cannot equal 1","If  Q2a. RACEWHITE, Q2b. RACEBLACK, Q2c. RACEAIAN, Q2d. RACENHPI, Q2e. RACEASIAN, Q2f. ETHISPANIC, or Q2g. RACEOTH=1 then Q2h. RACEUNKN cannot equal 1",IF (RACEWHITE=1 or RACEBLACK=1 or RACEAIAN=1 or RACENHPI=1 or RACEASIAN=1 or ETHISPANIC=1 or RACEOTH=1) and RACEUNKN=1,,"RACEWHITE,
```

Comment:
Please change the var_name for error a1-ivp-p-1001 to "RACEUNKN", it's easy to implement this check under that variable.